### DISCUSSION TITLE
Should /admin but accessible even when KnockKnock is enabled?

### DISCUSSION THREAD

**Discussion by RyanRoberts** [verbb/knock-knock#57] [Q&A]
I've used KK for ages without issue but for some reason I thought if you have a user account you can bypass KK and just go straight to the CP admin and login without entering the KK password. But at the moment KK is redirecting me to the locked page whenever I try to access the CP.

I've added /admin/(.*) to the unprotected URLS and I can access it but it gives me an undefined error.

The reason I'm asking this is because I have KK acting as a holding page *without* the password field, I was hoping to then login and access the site as normal.

**Comment by engram-design** [comment]
Transferring this to an issue, as I believe this is a bug https://github.com/verbb/knock-knock/issues/58 in the latest version.
